### ISSUE TITLE
[Feature Request] A2C

### ISSUE THREAD

**Issue by vmoens** [pytorch/rl#17]
Implement A2C algorithm as presented [here](https://arxiv.org/abs/1602.01783)

**Comment by sarang7m** [comment]
can i solve this issue and send a PR ?

**Comment by vmoens** [comment]
Of course, any contribution is welcome!
It's probably a bit tedious as a first task but why not :) 
We'd need the following things
- a training script in `examples/a2c/a2c.py`
- a dedicated loss function (if no other can be used)
- helper functions to make the model and build the loss function
Don't hesitate to reach out if you need help!
